Makefile

```diff
@@ -85,10 +85,10 @@ lint-server: test-dependencies
 	flake8 elyra
 
 prettier-ui:
-	yarn prettier:check
+	yarn prettier
 
 eslint-ui:
-	yarn eslint:check --max-warnings=0
+	yarn eslint
```

Comment:
is there a reason we are changing this?

Reply:
yeah, I'm unsure why this was changed, I would've not approved it myself, it removes functionality that was intentional. Changing this to check just makes lint fail with no help to the user. Prettier is meant to fix and the fix flag on eslint only make minor intentional changes, the rest are still just displayed, though on a re-read I should leave `--max-warnings=0` as that is useful to add. The use of fix here matches how jupyterlab lint functions

Reply:
Maybe we need to distinguish between what's run on ci vs locally? I don't think that eslint/prettier should stop a local build. On ci we want it to fail, if changes need to be made to the code